egs/aishell/asr/simple_v1/RESULTS.md

```diff
@@ -1,10 +1,17 @@
 # AIShell MMI training results
 
 ## 2021-02-10
-(Pingfeng Luo): This CER is obtained using the latest k2 and lhotse as of today (2021-02-10).
+(Pingfeng Luo):
+This CER is obtained using the latest k2 and lhotse at 2021-02-10.
 ```
 # epoch 7
-2021-02-10 18:12:22,691 INFO [mmi_bigram_decode.py:263] %WER 10.06% [10542 / 104765, 436 ins, 495 del, 9611 sub ] exp-lstm-adam-mmi-bigram-musan
+2021-02-10 18:12:22,691 INFO [mmi_bigram_decode.py:263] %WER 10.06% [10542 / 104765, 436 ins, 495 del, 9611 sub ]
+```
+## 2021-03-11
+(Pingfeng Luo):
+following CER is average over epochs 10 to 14, using mmi_att_conformer model.
```

Comment:
The way you write it makes it sound like the CER is averaged, rather than the model parameters.

Reply:
plus it would be nice to have some diagnostics available so people can easily compare to the baseline.
Could include some selected lines of output in the RESULTS.md, or an upload of the tensorboard log.

Reply:
will fix it.

Reply:
i try to upload tensorboard log but there is  a err, maybe because my network can't access googleclound.

Reply:
mm yes, you may need to set some kind of proxy from the command line but I dont know how..
Perhaps just grep for epoch-9 (assuming 10 epochs) and show that line.

Reply:
cat i just put these log to  RESULTS.md？
```
cat epoch-5-info epoch-6-info epoch-7-info epoch-8-info epoch-9-info 
model_path: exp-conformer-noam-mmi-att-musan/epoch-5.pt
epoch: 5
learning rate: 0.00029682922689692776
objf: 0.09375367592695481
best objf: 0.09375367592695481
valid objf: 0.09187789649726409
best valid objf: 0.09187789649726409
best epoch: 5
model_path: exp-conformer-noam-mmi-att-musan/epoch-6.pt
epoch: 6
learning rate: 0.0002700218618549395
objf: 0.0880647696924415
best objf: 0.0880647696924415
valid objf: 0.09059886088615578
best valid objf: 0.09059886088615578
best epoch: 6
model_path: exp-conformer-noam-mmi-att-musan/epoch-7.pt
epoch: 7
learning rate: 0.00024937237144202914
objf: 0.08386929301927937
best objf: 0.0880647696924415
valid objf: 0.09063070541871378
best valid objf: 0.09059886088615578
best epoch: 6
model_path: exp-conformer-noam-mmi-att-musan/epoch-8.pt
epoch: 8
learning rate: 0.000232831603379837
objf: 0.08086437557875469
best objf: 0.08386929301927937
valid objf: 0.09076970567486643
best valid objf: 0.09063070541871378
best epoch: 8
model_path: exp-conformer-noam-mmi-att-musan/epoch-9.pt
epoch: 9
learning rate: 0.00021919976614813435
objf: 0.0785176101977766
best objf: 0.08086437557875469
valid objf: 0.09090691501700747
best valid objf: 0.09076970567486643
best epoch: 9
```